src/python/TaskWorker/Actions/Recurring/TapeRecallStatus.py

```diff
@@ -6,13 +6,15 @@
 from RESTInteractions import HTTPRequests
 from TaskWorker.Actions.MyProxyLogon import MyProxyLogon
 from TaskWorker.WorkerExceptions import TaskWorkerException
+from ServerUtilities import getTimeFromTaskname
 
 import logging
 import sys
-import os
+import os, time
 
 class TapeRecallStatus(BaseRecurringAction):
     pollingTime = 60*4 # minutes
+    maxLifeDays = 30
```

Comment:
I'd rather have such constance in ServerUtilities like the current
`TASKLIFETIME = 30*24*60*60`